container-engine-lib/lib/backend_impls/remote_context_backend/remote_context_kurtosis_backend.go

```diff
@@ -0,0 +1,208 @@
+	return backend.remoteKurtosisBackend.GetNetworkingSidecars(ctx, filters)
+}
+
+func (backend *RemoteContextKurtosisBackend) RunNetworkingSidecarExecCommands(ctx context.Context, enclaveUuid enclave.EnclaveUUID, networkingSidecarsCommands map[service.ServiceUUID][]string) (successfulNetworkingSidecarExecResults map[service.ServiceUUID]*exec_result.ExecResult, erroredUserServiceUuids map[service.ServiceUUID]error, resultErr error) {
+	return backend.remoteKurtosisBackend.RunNetworkingSidecarExecCommands(ctx, enclaveUuid, networkingSidecarsCommands)
+}
+
+func (backend *RemoteContextKurtosisBackend) StopNetworkingSidecars(ctx context.Context, filters *networking_sidecar.NetworkingSidecarFilters) (successfulUserServiceUuids map[service.ServiceUUID]bool, erroredUserServiceUuids map[service.ServiceUUID]error, resultErr error) {
+	return backend.remoteKurtosisBackend.StopNetworkingSidecars(ctx, filters)
+}
+
+func (backend *RemoteContextKurtosisBackend) DestroyNetworkingSidecars(ctx context.Context, filters *networking_sidecar.NetworkingSidecarFilters) (successfulUserServiceUuids map[service.ServiceUUID]bool, erroredUserServiceUuids map[service.ServiceUUID]error, resultErr error) {
+	return backend.remoteKurtosisBackend.DestroyNetworkingSidecars(ctx, filters)
+}
+
+func (backend *RemoteContextKurtosisBackend) CreateLogsDatabase(ctx context.Context, logsDatabaseHttpPortNumber uint16) (*logs_database.LogsDatabase, error) {
+	return backend.remoteKurtosisBackend.CreateLogsDatabase(ctx, logsDatabaseHttpPortNumber)
+}
+
+func (backend *RemoteContextKurtosisBackend) GetLogsDatabase(ctx context.Context) (*logs_database.LogsDatabase, error) {
+	return backend.remoteKurtosisBackend.GetLogsDatabase(ctx)
+}
+
+func (backend *RemoteContextKurtosisBackend) DestroyLogsDatabase(ctx context.Context) error {
+	return backend.remoteKurtosisBackend.DestroyLogsDatabase(ctx)
+}
+
+func (backend *RemoteContextKurtosisBackend) CreateLogsCollectorForEnclave(ctx context.Context, enclaveUuid enclave.EnclaveUUID, logsCollectorHttpPortNumber uint16, logsCollectorTcpPortNumber uint16) (*logs_collector.LogsCollector, error) {
+	return backend.remoteKurtosisBackend.CreateLogsCollectorForEnclave(ctx, enclaveUuid, logsCollectorHttpPortNumber, logsCollectorHttpPortNumber)
+
+}
+
+func (backend *RemoteContextKurtosisBackend) GetLogsCollectorForEnclave(ctx context.Context, enclaveUuid enclave.EnclaveUUID) (*logs_collector.LogsCollector, error) {
+	return backend.remoteKurtosisBackend.GetLogsCollectorForEnclave(ctx, enclaveUuid)
+}
+
+func (backend *RemoteContextKurtosisBackend) DestroyLogsCollectorForEnclave(ctx context.Context, enclaveUuid enclave.EnclaveUUID) error {
+	return backend.remoteKurtosisBackend.DestroyLogsCollectorForEnclave(ctx, enclaveUuid)
+}
+
+func (backend *RemoteContextKurtosisBackend) DestroyDeprecatedCentralizedLogsResources(ctx context.Context) error {
```

Comment:
tangent - perhaps we should delete this completely now, we have released a few versions since this